src/mjolnir/pbfgraphparser.cc

```diff
@@ -106,10 +106,12 @@ struct graph_callback : public OSMPBF::Callback {
       return;
     }
 
-    // Get tags if not already available
-    results = results ? results : lua_.Transform(OSMType::kNode, tags);
-    if (results->size() == 0) {
-      return;
+    // Get tags if not already available.  Don't bother calling Lua if there
+    // are no OSM tags to process.
+    if (tags.size() > 0) {
```

Comment:
maybe want this logic on line 90 as well as it pretty much nukes this change if someone has turned on bike share parsing